src/main/java/edu/hm/hafner/coverage/parser/CoberturaParser.java

```diff
@@ -140,17 +141,20 @@ else if (event.isEndElement()) {
 
     private FileNode createFileNode(final StartElement element, final PackageNode packageNode) {
         var fileName = getValueOf(element, FILE_NAME);
-        var path = getTreeStringBuilder().intern(PATH_UTIL.getRelativePath(fileName));
-
-        return packageNode.findOrCreateFileNode(getFileName(fileName), path);
+        var relativePath = PATH_UTIL.getRelativePath(fileName);
+        var actualPath = relativePath.startsWith("/_/") ? 
```

Comment:
Extract `"/_/"` as a constant with a meaningful name (deterministic...).

Reply:
can't you use `String#replaceFirst` here?

Reply:
Good point! I've replaced the manual prefix check with String#replaceFirst to simplify the logic